orga/app/services/locale.js

```diff
@@ -4,10 +4,10 @@ import config from 'pix-orga/config/environment';
 export const FRENCH_INTERNATIONAL_LOCALE = 'fr';
 export const ENGLISH_INTERNATIONAL_LOCALE = 'en';
 export const FRENCH_FRANCE_LOCALE = 'fr-FR';
-export const DEFAULT_LOCALE = FRENCH_INTERNATIONAL_LOCALE;
+export const DEFAULT_LOCALE = 'fr';
+const supportedLanguages = [FRENCH_INTERNATIONAL_LOCALE, ENGLISH_INTERNATIONAL_LOCALE];
```

Comment:
Uniquement si on veut laisser cette constante ici, alors on devrait faire cette modification :

```suggestion
const SUPPORTED_LANGUAGES = [FRENCH_INTERNATIONAL_LOCALE, ENGLISH_INTERNATIONAL_LOCALE];
```

Sinon elle peut reprendre la position qu'elle avait avant modification.

Reply:
J'ai mis le :+1: pour que dire que j'étais pour le fait de mettre `const SUPPORTED_LANGUAGES = [FRENCH_INTERNATIONAL_LOCALE, ENGLISH_INTERNATIONAL_LOCALE];` en haut du fichier.

Reply:
Je modifie !